docs/api/training-data-importers.rst

```diff
@@ -51,6 +54,88 @@ configuration file:
    importers:
    - name: "RasaFileImporter"
 
+SkillSelector (experimental)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+    This feature is currently experimental and might change or be removed in the future.
+    Please share your feedback on it in the `forum <https://forum.rasa.com>`_ to help
+    us making this feature ready for production.
+
+With this importer you can build a contextual AI assistant by combining
+reusable "building blocks" called skills.
+You might, for example, handle chitchat with one skill and greet your users with
+another. These skills can be developed in isolation, and then combined at train time
+to create your assistant.
+
+An example directory structure could look like this:
+
+.. code-block:: bash
+
+    .
+    ├── config.yml
+    └── skills
+        ├── GreetBot
+        │   ├── data
+        │   │   ├── nlu.md
+        │   │   └── stories.md
+        │   └── domain.yml
+        └── ChitchatBot
+            ├── config.yml
+            ├── data
+            │   ├── nlu.md
+            │   └── stories.md
+            └── domain.yml
+
+In this example the contextual AI assistant imports the ``ChitchatBot`` skill which in turn
+imports the ``GreetBot`` skill. Skill imports are defined in the configuration files of
+each project.
+To instruct Rasa to use the ``SkillSelector`` module, put this section in the config
+file of your root project:
+
+.. code-block:: yaml
+
+    importers:
+    - name: SkillSelector
+
+
+Then specify which skills you want to import.
+In our example, the ``config.yml`` in the root project would look like this:
+
+.. code-block:: yaml
+
+    imports:
+    - skills/ChitchatBot
+
+The configuration file of the ``ChitchatBot`` in turn references the ``GreetBot``:
+
+.. code-block:: yaml
+
+    imports:
+    - ../GreetBot
+
+The ``GreetBot`` skill does not specify further skills so the ``config.yml`` can be
+omitted.
+
+Rasa uses relative paths from the referencing configuration file to import skills.
+These can be anywhere on your file system as long as the file access is permitted.
+
+During the training process Rasa will import all required training files, combine
+them, and train a unified AI assistant.
+
+.. note::
+
+    Rasa will use the policy and NLU pipeline configuration of the root project
+    directory during the training. Policy or NLU configurations of imported skills will
+    be ignored.
```

Comment:
in bold:
**The configuration (NLU pipeline and policies) of imported projects will be ignored**